schemas/context/experienceevent.schema.json

```diff
@@ -126,6 +126,31 @@
           "description": "The media activity information related to the experience event"
         }
       },
+      "xdm:profileStitching" : {
+        "title": "Profile Stitching",
```

Comment:
This seems like a very Adobe-centric thing, yes?  If so, it belongs in an extension and *not* in the primary schema

Reply:
I don't think so. It's about which set of profiles segmentation is running against. Adobe has some functionality around this, but I think others in the industry do too.

Reply:
well, I would want to understand how it fits into the larger industry context before accepting this in a common schema.

Reply:
Understood, should we talk offline? Or do you have something in particular you're looking for?

Reply:
Certainly happy to chat offline, but what I am looking for is either an existing standard for this that we can point to from this common schema...or we move it to an Adobe extension.

Reply:
@cdegroot-adobe if I do a web search on "profile stitching", only one of the hits is about this (and it actually calls it visitor stitching).
Is there any existing industry reference document that we can point to that defines this?